rudder-lang/src/bin/main.rs

```diff
@@ -75,32 +91,47 @@ struct Opt {
     #[structopt(long, short)]
     input: PathBuf,
     /// Set to use technique translation mode
-    #[structopt(long)]
+    #[structopt(long, short)]
     translate: bool,
     /// Set to compile a single technique
-    #[structopt(long)]
-    technique: bool,
-    /// Output format to use
-    #[structopt(long, short = "f")]
-    output_format: Option<String>,
+    #[structopt(long, short)]
+    compile: bool,
```

Comment:
there are 2 options: 
one to separate compilation and translation
one to separate full compilation and single technique compilation